app/(dashboard)/page.tsx

```diff
@@ -47,6 +47,7 @@ export default async function ProductsPage({
           products={products}
           offset={newOffset ?? 0}
           totalProducts={totalProducts}
+          showAll={search !== ''}
```

Comment:
IMHO, you should still have pagination when searching.

Reply:
Yeah, I was not sure if it still should have pagination or not.
Was a bit confused by the `// Always search the full table, not per page` comment in the db.ts file 😅